srcs/incs/HttpRequestHandler.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef SRCS_INCS_HTTPREQUESTHANDLER_HPP_
+#define SRCS_INCS_HTTPREQUESTHANDLER_HPP_
+#include <string>
+#include <HttpRequest.hpp>
+#include <HttpResponse.hpp>
+#include <ServerConfig.hpp>
+
+class HttpRequestHandler {
```

Comment:
As mentioned, if you like the idea of using an interface, this class could inherit from it implementing the main interface methods that define a generic protocol handler interface, in this case `GetTawResponse` and `GetKeepAlive`

Reply:
I like this idea :100:. I've added IRequestHandler interface.